src/start.sh

```diff
@@ -27,20 +26,20 @@ echo "  [i] Starting docker specific checks & setup for docker pihole/pihole"
 
 # If PIHOLE_UID is set, modify the pihole user's id to match
 if [ -n "${PIHOLE_UID}" ]; then
-    currentId=$(id -u ${username})
-    [[ ${currentId} -eq ${PIHOLE_UID} ]] && return
-
+  currentId=$(id -u ${username})
+  if [[ ${currentId} -ne ${PIHOLE_UID} ]]; then
     echo "  [i] Changing ID for user: pihole (${currentId} => ${PIHOLE_UID})"
     usermod -o -u ${PIHOLE_UID} pihole
+  fi
```

Comment:
Actually, just for verbosity - can we add  something like

```bash
else
   echo "  [i] ID for user pihole is already ${PIHOLE_UID}, no need to change"
```

Might save some troubleshooting in the future.

Side note: I wonder if we should force the ids to be 999 by default as they are in the v5 containers. I noticed the other day that the id in the new container is 100 - which might cause issues with upgrades?

Reply:
I've pushed the unchanged info text.
As for the default uid/gid number, start uid depends on distribution, some will start a 100, other at 500, or 1000.
I'm using a ubuntu as host and a debian. I haven't experienced issues with uid/gid other than the "can't chmod <namethefile>". If it needs to be set, Dockerfile would be the good place to implement it when creation the user/group.